tests/core_tests/test_log.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+import copy
+
+import sgtk
+from tank_test.tank_test_base import *
```

Comment:
@boismej Any chance this is a policy that could be disabled for test files?

Reply:
The correct way to fix this is to to like we do in other test files:

```
from tank_test.tank_test_base import setUpModule # noqa
from tank_test.tank_test_base import TankTestBase
```

doing `import *` is bad because it then prevents tools like PyFlake from being able to tell that you are accessing an undefined variable. This is why `import *` is highly discouraged.

Reply:
# noqa will tell pycodestyle (aka pep8) to not emit an error on that line.

Reply:
Seems like you still haven't fixed the `import *`.